packages/angular_devkit/core/src/analytics/api.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+export interface CustomDimensionsAndMetricsOptions {
+  dimensions?: (boolean | number | string)[];
+  metrics?: (boolean | number | string)[];
+}
+
+export interface EventOptions extends CustomDimensionsAndMetricsOptions {
+  label?: string;
+  value?: string;
+  dimensions?: (boolean | number | string)[];
```

Comment:
I think you left in the inherited properties by accident, or maybe you meant to change them.

Reply:
That was a mistake, good catch!